workloads/kube-burner/kube-burner-crd.yaml

```diff
@@ -38,6 +38,12 @@ spec:
         VERIFY_OBJECTS: ${VERIFY_OBJECTS}
         ERROR_ON_VERIFY: ${ERROR_ON_VERIFY}
         MAX_WAIT_TIMEOUT: ${MAX_WAIT_TIMEOUT}
+        PPROF_COLLECTION: ${PPROF_COLLECTION}
+        PPROF_COLLECTION_INTERVAL: ${PPROF_COLLECTION_INTERVAL}
+        BEARER_TOKEN: ${BEARER_TOKEN}
+        KEY: ${KEY}
```

Comment:
Can you rename this one to PRIVATE_KEY?, I personally always try to avoid name variables KEY/key just in case is a reserved name and is a bit ambiguous.